tower-http/src/services/fs/serve_dir.rs

```diff
@@ -51,6 +53,15 @@ impl ServeDir {
         self.append_index_html_on_directories = append;
         self
     }
+    /// Cache-Control:max-age=300.
+    ///
+    /// This speeds up user's feeling for static sites, especially large files.
+    ///
+    /// Defaults to 300 seconds.
+    pub fn max_age(mut self, seconds: u64) -> Self {
```

Comment:
I think accepting a `std::time::Duration` is nicer and then mentioning the docs that it will be rounded down to the nearest second. Because thats what `Duration::as_secs` does.

```suggestion
    pub fn max_age(mut self, duration: Duration) -> Self {
```